target/linux/ramips/image/lzma-loader/src/board.c

```diff
@@ -0,0 +1,37 @@
+/*
+ * Arch specific code for ramips based boards
+ *
+ * Copyright (C) 2013 John Crispin <blogic@openwrt.org>
+ * Copyright (C) 2018 Tobias Schramm <tobleminer@gmail.com>
+ * Copyright (C) 2023 Antonio Vázquez <antoniovazquezblanco@gmail.com>
+ *
+ * This program is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 as published
+ * by the Free Software Foundation.
+ */
+
+#include <stdint.h>
+
+// Helper functions
+#define READREG(r)		(*(r))
+#define WRITEREG(r,v)	(*(r)) = v
+
+// Defined in board-$(SUBTARGET).c files
+extern uint32_t volatile * const UART_THR;
+extern uint32_t volatile * const UART_LSR;
+extern uint32_t const UART_LSR_THRE_MASK;
+
+// Generic initialization function. Can be overwritten in board-$(SUBTARGET).c
+// files if needed.
+void __attribute__((weak)) board_init(void)
+{
+}
+
+// Generic putc function. Can be overwritten in board-$(SUBTARGET).c files if
+// needed.
```

Comment:
I'd prefer a consistent usage of C style comments here.

Reply:
Easy! Will do! :)

Reply:
Done!

Reply:
Should I resolve this comments or should @blocktrron? What is your usual way to proceed? Thanks!